packages/common-server/src/errorReporting.ts

```diff
@@ -39,10 +40,29 @@ export function isBadErrorThatShouldBeSampled(
   );
 }
 
-export function initializeSentry(environment: Stage): void {
+/**
+ * Initialize Sentry
+ * @param environment
+ * @returns
+ *  ^4wcl13fw6gub
+ */
+export function initializeSentry({
+  environment,
+  sessionId,
+  release,
+}: {
+  environment: Stage;
+  sessionId?: number;
+  release: string;
+}): void {
   const dsn =
     "https://bc206b31a30a4595a2efb31e8cc0c04e@o949501.ingest.sentry.io/5898219";
 
+  const initialScope: CaptureContext = {};
+  if (sessionId) {
+    initialScope.tags = { sessionId };
```

Comment:
For future reference: if I wanted to add more properties to the user in Sentry (for example whether they have self contained vaults enabled or not), would this be where to add it?

Reply:
yep, just wrote docs as part of an async